feature/qos/otg_tests/mixed_sp_wrr_traffic_test/mixed_sp_wrr_traffic_test.go

```diff
@@ -738,6 +746,15 @@ func ConfigureQoS(t *testing.T, dut *ondatra.DUTDevice) {
 	q := d.GetOrCreateQos()
 	queues := netutil.CommonTrafficQueues(t, dut)
 
+	if dut.Vendor() == ondatra.NOKIA {
```

Comment:
Please use a deviation, instead of vendor switch. something like `qos_queue_requires_id`?

Reply:
Added new deviation `qos_queue_requires_id`.